README.md

```diff
@@ -53,19 +53,20 @@ github {
 
 where:
 
-  * `ip-address` is either an IPv4 or IPv6 address and `port` a port on which an
```

Comment:
Here the unrelated ip-address part changed due to an indent change. Was that intentional?

Reply:
Sort of. I found an indentation bug in the README while reformatting. I should probably have made a separate commit.

Reply:
That's ok. Just checking.